dropwizard-scala-parent-pom/pom.xml

```diff
@@ -0,0 +1,201 @@
+            </execution>
+            <execution>
+              <id>scala-test-compile-first</id>
+              <phase>process-test-resources</phase>
+              <goals>
+                <goal>testCompile</goal>
+              </goals>
+            </execution>
+          </executions>
+        </plugin>
+        <!-- enable scalatest -->
+        <plugin>
+          <groupId>org.scalatest</groupId>
+          <artifactId>scalatest-maven-plugin</artifactId>
+          <version>1.0</version>
+          <configuration>
+            <environmentVariables>
+              <SPARK_LOCAL_HOSTNAME>localhost</SPARK_LOCAL_HOSTNAME>
+            </environmentVariables>
+            <reportsDirectory>${project.build.directory}/surefire-reports
+            </reportsDirectory>
+            <junitxml>.</junitxml>
+            <filereports>WDF TestSuite.txt</filereports>
+          </configuration>
+          <executions>
+            <execution>
+              <id>test</id>
+              <goals>
+                <goal>test</goal>
+              </goals>
+            </execution>
+          </executions>
+        </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-release-plugin</artifactId>
+          <version>2.5.3</version>
+          <configuration>
+            <tagNameFormat>v@{project.version}</tagNameFormat>
+          </configuration>
+        </plugin>
+      </plugins>
+    </pluginManagement>
+  </build>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.mainstreethub.dropwizard.scala</groupId>
+        <artifactId>commons</artifactId>
+        <version>${dropwizard-scala.commons.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>com.datasift.dropwizard.scala</groupId>
+        <artifactId>dropwizard-scala-core_${scala_major.version}</artifactId>
+        <version>${dropwizard-scala.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>com.datasift.dropwizard.scala</groupId>
+        <artifactId>dropwizard-scala-jdbi_${scala_major.version}</artifactId>
+        <version>${dropwizard-scala.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>com.groupon.mysql</groupId>
+        <artifactId>mysql-junit4</artifactId>
+        <version>${mysql-junit4.version}</version>
+        <scope>test</scope>
+      </dependency>
+      <dependency>
+        <groupId>com.squareup.okhttp3</groupId>
+        <artifactId>mockwebserver</artifactId>
+        <version>${okhttp.version}</version>
+        <scope>test</scope>
+      </dependency>
+      <dependency>
+        <groupId>com.squareup.retrofit2</groupId>
+        <artifactId>retrofit</artifactId>
+        <version>${retrofit.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>com.squareup.retrofit2</groupId>
+        <artifactId>converter-jackson</artifactId>
+        <version>${retrofit.version}</version>
```

Comment:
You may want to confirm this pulls in the same versions of Jackson as Dropwizard and if not exclude them to prevent a downstream version conflict.